proposals/lip-0061.md

```diff
@@ -690,17 +711,18 @@ The function returns a valid certificate object that can be submitted to blockch
 For readability, we describe the logic using two functions, the main function `getNextCertificateFromSingleCommits` and the auxiliary function `computeEligibleSingleCommits`.
 
 ```python
-getNextCertificateFromSingleCommits(lastCertifiedHeight):
+def getNextCertificateFromSingleCommits(lastCertifiedHeight: uint32) -> Certificate:
     lastValidatorsHash  = validatorsHash property of block header at height lastCertifiedHeight
     lastCertifiedValidators = validatorsHashPreimage[lastValidatorsHash].validators
```

Comment:
`lastCertifiedValidators` is not used anymore

Reply:
Good point. After thinking about approach 2 and the required data more, I now only use the BFT parameters here and in the bullet point list about the required data, I am more explicit about the fact that past BFT parameters are needed.

Reply:
@ishantiw As this turned out to be a bit of a bigger change for this function and the required data, it would be nice if you could also have another look.

Reply:
can be resolved from my side. But I leave it open for @ishantiw

Reply:
Looks good to me, looks more simplified now